tests/vision/test_markers.py

```diff
@@ -0,0 +1,136 @@
+"""Tests for the Marker Class."""
+
+from math import isclose
+
+from j5.vision.markers import Coordinate, Cylindrical, Marker, Spherical
+
+
+def assert_coordinate_isclose(coord1: Coordinate, coord2: Coordinate):
```

Comment:
Just a thought: I wonder if creating `Cartesian.isclose`/`Cylindrical.isclose`/`Spherical.isclose` methods would be useful to users. Then this function could be replaced with simply `assert coord1.cartesian.isclose(coord2.cartesian)`.

Reply:
See #341 8ae1d3c where I have implemented `Coordinate.isclose`